i18n/zh/docusaurus-plugin-content-docs/current/advanced/debug.md

```diff
@@ -1,71 +1,70 @@
 ---
-title: 启用Kubectl logs/exec/attach等能力
+title: 启用 Kubectl logs/exec/attach 边端运行的 pod
```

Comment:
```suggestion
title: 启用 Kubectl logs/exec/attach 操作边缘 pods
```

Reply:
Thank you for your advice, I have made the corresponding changes.